README.md

```diff
@@ -51,6 +51,18 @@ PRs merged to the `master` branch are automatically deployed to staging.
 Production is automatically deployed upon merges to `release`. Create such a PR
 with [deploy_pr][deploy_pr] or [this handy link][deploy].
 
+## API
+
+Trusted apps may use the Bearden API using [JWT](https://jwt.io/) authentication. You will need to obtain the JWT secret for Bearden, please ask a Bearden developer for this. Once in possession of the secret, you need to encode a JWT token using the [JWT gem](https://github.com/jwt/ruby-jwt):
```

Comment:
I would suggest to use JWT created by Gravity in this case too. Thats what we do in Impulse as well, in that scenario `JWT_SECRET` is basically `internal_secret` of that `ClientApplication` in Gravity. In order to create a new JWT, then we can refer to [this](https://github.com/artsy/gravity/blob/master/doc/ApiAuthentication.md#create-xapp-token) doc.